strawberryfields/configuration.py

```diff
@@ -14,139 +14,214 @@
 r"""
 This module contains the :class:`Configuration` class, which is used to
 load, store, save, and modify configuration options for Strawberry Fields.
+
+.. warning::
+
+    The following configuration options are available:
+    :doc:`/introduction/configuration`.
```

Comment:
For consistency with our other developer pages:
```rest
.. warning::

    Unless you are a Strawberry Fields developer, you likely do not need to access this module directly.
    See more details regarding Strawberry Fields configuration and available configuration options 
    on the :doc:`/introduction/configuration` page.
```

Reply:
Sure! Here the first part will already be included from https://github.com/XanaduAI/strawberryfields/blob/demo/doc/code/sf_configuration.rst

so I'll add the reworded second part :)